sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/CreateUpdateOptions.cs

```diff
@@ -53,6 +53,7 @@ public CreateUpdateOptions(IDictionary<string, string> additionalProperties = de
         /// Gets or sets unmatched properties from the message are deserialized
         /// this collection
         /// </summary>
+        [JsonExtensionData]
```

Comment:
Run the tests on your local machine, see if they pass, if they fail with a error related to jsonextension data, if they do, remove this.

Reply:
Removed the line. Thanks.